packages/trader/src/Modules/Contract/Components/LastDigitPrediction/last-digit-prediction.tsx

```diff
@@ -1,11 +1,28 @@
 import classNames from 'classnames';
 import { observer } from 'mobx-react';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { isMobile } from '@deriv/shared';
-import DigitDisplay from './digit-display.jsx';
-import LastDigitPointer from './last-digit-pointer.jsx';
+import DigitDisplay from './digit-display';
+import LastDigitPointer from './last-digit-pointer';
+import { ProposalOpenContract, TicksStreamResponse } from '@deriv/api-types';
 
+type TLastDigitPrediction = {
+    barrier: React.ComponentProps<typeof DigitDisplay>['barrier'];
+    contract_type?: string;
+    digits: number[];
+    digits_info: { [key: number]: { digit: number; spot: string } };
+    dimension: number;
+    is_digit_contract?: React.ComponentProps<typeof DigitDisplay>['is_digit_contract'];
```

Comment:
Isn't it boolean? I mean `is_digit_contract`  and `has_entry_spot`. Please ignore, if it's not relevant.

Reply:
Correct. they defined in the child of this component(`DigitDisplay`) as `boolean`.